UnitySDK/Assets/ML-Agents/Scripts/DemonstrationRecorder.cs

```diff
@@ -10,6 +10,7 @@ namespace MLAgents
     /// Demonstration Recorder Component.
     /// </summary>
     [RequireComponent(typeof(Agent))]
+    [AddComponentMenu("ML Agents/Demonstration Recorder")]
```

Comment:
Should we list this? I'd be willing to remove it.

Reply:
I think it makes sense

Reply:
Makes sense to have it or remove it?